src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,49 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
     public void getStatistic(String fromFileName, String toFileName) {
+        writeToFile(toFileName, createReport(fromFileName));
+    }
 
+    private String createReport(String fromFileName) {
+        int supplyTotal = 0;
+        int buyTotal = 0;
+        String line;
+        StringBuilder builder = new StringBuilder();
+        try (BufferedReader inputFile = new BufferedReader(new FileReader(fromFileName))) {
+            while ((line = inputFile.readLine()) != null) {
+                String[] splittedLine = line.split(",");
+                if (splittedLine[0].equals("supply")) {
+                    supplyTotal += Integer.parseInt(splittedLine[1]);
+                } else if (splittedLine[0].equals("buy")) {
+                    buyTotal += Integer.parseInt(splittedLine[1]);
+                }
+            }
```

Comment:
You should insert `catch` block here. Close your `try` as soon as possible, don't create one `try` for a whole program logic.